### PR TITLE
Fix area effect cloud

### PR DESCRIPTION
**Pull request by ForceUpdate1**
add 1.14 particle type

### CODE REVIEW

File: common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_14.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import us.myles.ViaVersion.api.minecraft.metadata.Metadata;
 import us.myles.ViaVersion.api.type.Type;
+import us.myles.ViaVersion.api.type.types.minecraft.Particle1_14Type;
+import us.myles.ViaVersion.protocols.protocol1_13to1_12_2.data.Particle;
```
Comment (creeper123123321) [inline common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_14.java]: This could be moved into api package in abstraction branch

**Comment by ForceUpdate1** [comment]
I moved all particle types into the api package

File: common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_13_2.java [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 import us.myles.ViaVersion.api.minecraft.metadata.Metadata;
 import us.myles.ViaVersion.api.type.Type;
+import us.myles.ViaVersion.api.type.types.Particle;
+import us.myles.ViaVersion.protocols.protocol1_13_2to1_13_1.types.Particle1_13_2Type;
```
Comment (creeper123123321) [inline common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_13_2.java]: This isn't in api package

File: common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_13.java [hunk 3]
```diff
@@ -3,6 +3,8 @@
 import us.myles.ViaVersion.api.minecraft.chunks.ChunkSection;
 import us.myles.ViaVersion.api.minecraft.metadata.Metadata;
 import us.myles.ViaVersion.api.type.Type;
+import us.myles.ViaVersion.api.type.types.Particle;
+import us.myles.ViaVersion.protocols.protocol1_13to1_12_2.types.Particle1_13Type;
```
Comment (creeper123123321) [inline common/src/main/java/us/myles/ViaVersion/api/type/types/version/Types1_13.java]: Same here

**Comment by ForceUpdate1** [comment]
There are some other types that are not in the api package, like the chunk type. I moved the global variable and the abstract particle type.

**Comment by ForceUpdate1** [comment]
https://github.com/ViaVersion/ViaVersion/blob/3b97f843cd505dd83ac69f4ea7ce18d00a5be4bb/common/src/main/java/us/myles/ViaVersion/protocols/protocol1_13to1_12_2/types/Chunk1_13Type.java#L22

**Comment by MylesIsCool** [comment]
Ty

**Pull request merged**